shell/app/modules/apiManagePlatform/pages/api-market/design/index.tsx

```diff
@@ -575,7 +575,7 @@ const ApiDesign = () => {
                             </WithAuth>
                           )
                           :
-                            <Button type="ghost" disabled={formErrorNum > 0} onClick={commitSaveApi} >{i18n.t('save')}</Button>
+                            <Button type="ghost" disabled={formErrorNum > 0} onClick={() => commitSaveApi()} >{i18n.t('save')}</Button>
```

Comment:
what's the difference？ store seems won't take event as payload

Reply:
It is best to explicitly indicate that no parameters are required